storage/grpc_client.go

```diff
@@ -861,13 +861,6 @@ func (c *grpcStorageClient) NewRangeReader(ctx context.Context, params *newRange
 		ctx = setUserProjectMetadata(ctx, s.userProject)
 	}
 
-	// A negative length means "read to the end of the object", but the
```

Comment:
Is there a reason to make this gRPC changes in the same PR? it might make more sense to do them separately unless necessary.

Reply:
Good point. Originally it was to fix part of the test for GRPC, but we are skipping it anyway. Separate PR opened.